fluentd/configs.d/openshift/filter-syslog-record-transform.conf

```diff
@@ -37,6 +37,7 @@
     # Plus we'd better avoid the file IO.
     hostname ${_HOSTNAME.eql?('localhost') ? (begin; File.open('/etc/docker-hostname') { |f| f.readline }.rstrip; rescue; _HOSTNAME; end) : _HOSTNAME}
     message ${record["MESSAGE"]}
+    level ${record["PRIORITY"]}
```

Comment:
I believe [Anton](https://github.com/t0ffel/fluentd-openshift/blob/3b2098c3bb4e722bc7a161648b17fedf865db46a/configs/node-collector/journal/filter-k8s-record-transform.conf) has a fix in progress for that.